force-app/main/default/classes/TriggerHandler.cls

```diff
@@ -0,0 +1,240 @@
+public virtual class TriggerHandler {
```

Comment:
Well done implementing the TriggerHandler framework that you worked with in the homeworks!